superchain/superchain.go

```diff
@@ -187,6 +188,8 @@ func (c *ChainConfig) setNilHardforkTimestampsToDefaultOrZero(s *SuperchainConfi
 // EnhanceYAML creates a customized yaml string from a RollupConfig. After completion,
 // the *yaml.Node pointer can be used with a yaml encoder to write the custom format to file
 func (c *ChainConfig) EnhanceYAML(ctx context.Context, node *yaml.Node) error {
+	ethAddressRegex := regexp.MustCompile(`^0x[a-fA-F0-9]+$`)
```

Comment:
Nit: maybe we should call this "hexStringRegex"? It will match the genesis hash, which is not an address.

Reply:
Good catch. Will push a fix